src/wrappers.cpp

```diff
@@ -141,7 +141,7 @@ realToVirtualPath(char *path)
 
   PluginManager::eventHook(DMTCP_EVENT_REAL_TO_VIRTUAL_PATH, &data);
 
-  return path;
+  return data.realToVirtualPath.path;
```

Comment:
For real to virtual conversion, the `path` is updated in-place so at the end of the function, `data.realtoVirtualPath.path` and `path` point to the same location.

Reply:
Is there any particular reason for this change?

Reply:
`data` is an in and out parameter so that it's possible to change where `data.realtoVirtualPath.path` points to. So we think it could be more stable this way. Also, the `realtoVirtialPah` returns the result with both the `path` and return value, but the return value is never used. Can we simplify this?

Reply:
We don't use the return value right now, but I can imagine situations similar to `virtualToRealPath`. The idea was to keep the two signatures similar.

Re. `data` being an in/out parameter, it's a good point. Let's return `data.realToVirtualPath.path` instead of `path`.